### DISCUSSION TITLE
I have some old windows server OS version that cant install? Any solution?

### DISCUSSION THREAD

**Discussion by mutallim** [slackhq/nebula#819] [Q&A]
Windows server 2012 and 2012 R
Windows server 2008
cant install windows client application to those server versions.

**Comment by johnmaguire** [comment]
@mutallim Are you able to provide any additional information regarding the problems you ran into? What did you try, what happened, what did you expect, etc? Thanks.

**Reply by JuanZer0** [reply]
I don't know if is the same case but I am getting an error when the interface is going up, this is on windows server 2008 r2

time="2023-04-26T13:55:17-06:00" level=fatal msg="failed to set address: El objeto ya existe." 

the spanish part says "object already exists"

This error remains after reboot it appears something is not being flushed out
